Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -67,25 +69,17 @@
   - type: Clothing
     sprite: _NF/Objects/Weapons/Guns/Rifles/gestio.rsi
   - type: Gun
-    fireRate: 3.75
-    projectileSpeed: 30
-    angleDecay: 4 #in testing 3 was more balanced
-    angleIncrease: 6
-    minAngle: 21 #generally meant to be 1 when used properly
-    maxAngle: 30 #generally meant to be 5 when used properly
+    fireRate: 5
+    burstFireRate: 5
+    shotsPerBurst: 2
+    burstCooldown: .33
```

```suggestion
    fireRate: 3.8
    burstFireRate: 8
    shotsPerBurst: 2
    burstCooldown: 0.4
```
3.8 kept for description (or if you want a semi-auto mode), should be accurate overall.